### DISCUSSION TITLE
Why is there no new "Who is hiring/wants to be hired" thread for two months?

### DISCUSSION THREAD

**Discussion by boristadic** [vercel/next.js#52379] [Help] [answered]
### Summary

Last thread is from May, is this idea abandoned or jobs aren't important any more or job market is completely dead?

https://github.com/vercel/next.js/discussions/49471

**Comment by Nutlope** [comment, marked as answer]
Just posted the job threads for this month, thanks for the reminder!

https://github.com/vercel/next.js/discussions/52399
https://github.com/vercel/next.js/discussions/52398
